lib/dpul_collections/solr/constants.ex

```diff
@@ -39,6 +39,34 @@ defmodule DpulCollections.Solr.Constants do
           label: Gettext.Macros.gettext_with_backend(DpulCollectionsWeb.Gettext, "Genre"),
           # Identity just returns whatever you gave it.
           value_function: &Function.identity/1
+        },
+        "project" => %{
+          solr_field: "ephemera_project_title_s",
+          label: Gettext.Macros.gettext_with_backend(DpulCollectionsWeb.Gettext, "Project"),
```

Comment:
I think I called it "Ephemera project" in the metadata pane. But maybe you're trying to be more terse here?

Reply:
Yeah it just seemed like a long facet label, but happy to keep it consistent for now.